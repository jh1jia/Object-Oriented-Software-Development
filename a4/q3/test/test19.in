test16.txt count 5 dropfirst 0 count 3 count 4
