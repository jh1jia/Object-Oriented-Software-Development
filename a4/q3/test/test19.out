1111111111
2222222222
12315678910
01211216789
98762312210
78610921364
1
2
15
6
15
6
7
0
9
